CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [FEATURE] Querier: Queries can be federated across multiple tenants. The tenants can IDs involved need to be specified separated by a `|` character in the `X-Scope-OrgID` header. This is an experimental feature, which can be enabled by setting `-tenant-federation.enabled` on all components. #3250
```

Comment:
```suggestion
* [FEATURE] Querier: Queries can be federated across multiple tenants. The tenants IDs involved need to be specified separated by a `|` character in the `X-Scope-OrgID` request header. This is an experimental feature, which can be enabled by setting `-tenant-federation.enabled=true` on all Cortex services. #3250
```

Reply:
Thanks for that suggestion. Has been added.